src/Apps/Auctions/Components/MyBids/MyBidsBidItem.tsx

```diff
@@ -103,14 +103,16 @@ export const MyBidsBidItem: React.FC<
                   </Text>
                 )}
 
-                {saleArtwork.sale?.isLiveOpen ? (
+                {saleArtwork.sale?.isLiveOpenHappened ? (
                   <Text
                     variant="xs"
-                    color="blue100"
+                    color={saleArtwork.sale?.isLiveOpen ? "blue100" : "mono60"}
                     overflowEllipsis
                     display="flex"
                   >
-                    &nbsp; Bidding live now
+                    {saleArtwork.sale?.isLiveOpen
```

Comment:
If we know it's in live bidding - display that, otherwise just say that `Live auction` in grey to keep it generic